src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -104,6 +104,7 @@ EmbeddedAssemblies::get_assembly_data (uint8_t *data, uint32_t data_size, [[mayb
 			if (cad.loaded) {
 				set_assembly_data_and_size (reinterpret_cast<uint8_t*>(cad.data), cad.uncompressed_file_size, assembly_data, assembly_data_size);
 				return;
+			}
```

Comment:
glad I'm not the only one who makes these kinds of typo's 🤣 👍

Reply:
:D manual merge conflict resolution FTW :)